modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -404,6 +415,7 @@ object ElasticRequest {
     from: Option[Int],
     highlights: Option[Highlights],
     routing: Option[Routing],
+    searchAfter: Option[Json],
     size: Option[Int]
   ) extends SearchAndAggregateRequest { self =>
     def from(value: Int): SearchAndAggregateRequest =
```

Comment:
```
 def highlights(value: Highlights): SearchAndAggregateRequest =
      self.copy(highlights = Some(value))
```

Please provide this method in `sealed trait SearchAndAggregateRequest`
    def highlights(value: Highlights): SearchAndAggregateRequest